examples/01_plotting/plot_demo_more_plotting.py

```diff
@@ -31,7 +31,7 @@
 from nilearn import datasets
 
 # haxby dataset to have anatomical image, EPI images and masks
-haxby_dataset = datasets.fetch_haxby(n_subjects=1)
+haxby_dataset = datasets.fetch_haxby(subjects=[1])
```

Comment:
Let's use subject 2 here (or simply use the default, and hence leave out the argument).